### PR TITLE
Keyword completion improvements and plugin update to 1.1.0

### PR DESCRIPTION
**Pull request by RAVEENSR**
## Purpose
This PR contains keyword completion improvements, updates for the siddhi grammar and plugin version upgrade.

## Goals
Enhance the user experience 

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: src/main/java/org/wso2/siddhi/plugins/idea/completion/executionelements/query/QueryOutputRateCompletionContributor.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mohanvive) [inline src/main/java/org/wso2/siddhi/plugins/idea/completion/executionelements/query/QueryOutputRateCompletionContributor.java]: Change the license year?

File: src/main/java/org/wso2/siddhi/plugins/idea/psi/AggregationNameNode.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (mohanvive) [inline src/main/java/org/wso2/siddhi/plugins/idea/psi/AggregationNameNode.java]: Fix license year.

Comment (mohanvive) [reply src/main/java/org/wso2/siddhi/plugins/idea/psi/AggregationNameNode.java]: @RAVEENSR check other places as well (where u have added new files)

Comment (RAVEENSR) [reply src/main/java/org/wso2/siddhi/plugins/idea/psi/AggregationNameNode.java]: Fixed in the commit aca3f1f

File: src/main/java/org/wso2/siddhi/plugins/idea/completion/executionelements/query/QueryOutputRateCompletionContributor.java (hunk 1 above)
Comment (RAVEENSR) [reply src/main/java/org/wso2/siddhi/plugins/idea/completion/executionelements/query/QueryOutputRateCompletionContributor.java]: Fixed in the commit aca3f1f

**Review by mohanvive — APPROVED** [review]

File: src/main/java/org/wso2/siddhi/plugins/idea/completion/definitionelements/DefinitionCompletionContributor.java [hunk 3]
```diff
@@ -15,8 +15,224 @@
...
+        // suggestions after an attribute type if it is in a definition element other than an aggregation definition
+        // if the aggregation is also included it will provide attribute types after the attributes in 'select'
+        // clause inside the aggregation definition
+        if (PsiTreeUtil.getParentOfType(prevVisibleSibling, AttributeNameNode.class) != null
+                && PsiTreeUtil.getParentOfType(prevVisibleSibling, DefinitionElementNode.class) != null
+                && PsiTreeUtil.getParentOfType(prevVisibleSibling, AggregationDefinitionNode.class) == null) {
+            addValueTypesAsLookups(result);
+            return;
+        }
+        //TODO: on that class handle attribute type suggestions
```
Comment (fazlan-nazeem) [inline src/main/java/org/wso2/siddhi/plugins/idea/completion/definitionelements/DefinitionCompletionContributor.java:82]: Is this comment redundant or this still needs to be done?

Comment (RAVEENSR) [reply src/main/java/org/wso2/siddhi/plugins/idea/completion/definitionelements/DefinitionCompletionContributor.java:82]: This is still needed to be done

**Pull request merged**